cnn_framework/molecule_tensor.py

```diff
@@ -203,7 +203,7 @@ def get_bond_attributes(molecule,
 
     if not bond_attributes_dict:
         if differentiate_bond_type:
-            bond_attributes_dict['no_bond'] = np.array([0]*15, dtype=np.float32)
+            bond_attributes_dict['no_bond'] = np.array([0]*14, dtype=np.float32)
```

Comment:
This commit can also be squashed with one of the previous ones.

Reply:
Done